packages/clay-color-picker/src/Editor.tsx

```diff
@@ -242,7 +265,11 @@ export function Editor({
 									}
 								}}
 								type="text"
-								value={hex.toUpperCase().substring(0, 6)}
+								value={
+									hex.toUpperCase().slice(-2) === 'FF'
+										? hex.toUpperCase().substring(0, 6)
```

Comment:
Maybe we need to change this here later as it now supports hexadecimal with alpha, maybe to 8.